Makefile

```diff
@@ -54,7 +54,7 @@ T_YELLOW := \e[0;33m
 T_RESET := \e[0m
 
 .PHONY: latest
-latest: 1.27 ## Build EKS Optimized AL2 AMI with the latest supported version of Kubernetes
+latest: 1.28 ## Build EKS Optimized AL2 AMI with the latest supported version of Kubernetes
```

Comment:
We should wait to change this default until 1.28 control planes are GA.

Reply:
fixed 👍